data/mods/Magiclysm/items/black_dragon_items.json

```diff
@@ -249,5 +249,182 @@
     "material_thickness": 2,
     "environmental_protection": 1,
     "flags": [ "VARSIZE", "WATERPROOF", "STURDY" ]
+  },
+  {
+    "id": "boots_xlblack_dragon_scale",
+    "copy-from": "boots_black_dragon_scale",
+    "type": "ARMOR",
+    "name": { "str": "pair of XL black dragonscale boots", "str_pl": "pairs of XL black dragonscale boots" },
+    "description": "Massive boots made of black dragonscale, modified to fit even the strangest of bodies. Very protective, and surprisingly light.",
```

Comment:
```suggestion
    "description": "Massive boots made of black dragonscale, modified to fit even the strangest of bodies.  Very protective, and surprisingly light.",
```

Reply:
i thought we were supposed to avoid double spaces?